assets/js/modules/adsense/components/dashboard/AdSenseConnectCTAWidget.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * AdSenseConnectCTAWidget component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import AdSenseConnectCTA from '../common/AdSenseConnectCTA';
+import { ADSENSE_CTA_WIDGET_DISMISSED_ITEM_KEY } from '../../constants';
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+const { useSelect } = Data;
+
+function AdSenseConnectCTAWidget( { Widget } ) {
+	const adSenseModuleConnected = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleConnected( 'adsense' )
+	);
+	const hasDismissedWidget = useSelect( ( select ) =>
+		select( CORE_USER ).isItemDismissed(
+			ADSENSE_CTA_WIDGET_DISMISSED_ITEM_KEY
+		)
+	);
+
+	if ( adSenseModuleConnected || hasDismissedWidget ) {
+		return null;
```

Comment:
I believe this needs to return the `WidgetNull` from props instead. Just noticed this is also off in the IB - widgets must never return `null`, in that situation it always has to be `WidgetNull`.

Reply:
👍